Barotrauma/BarotraumaShared/Source/Characters/StatusEffect.cs

```diff
@@ -272,6 +262,66 @@ protected StatusEffect(XElement element)
                         
                         requiredItems.Add(newRequiredItem);
                         break;
+                    case "conditional":
+                        IEnumerable<XAttribute> conditionalAttributes = subElement.Attributes();
+                        foreach(XAttribute attribute in conditionalAttributes)
+                        {
+                            string attributeString = XMLExtensions.GetAttributeObject(attribute).ToString();
+                            string atStr = attributeString;
+                            string[] splitString = atStr.Split(' ');
+                            string op = splitString[0];
+                            if (splitString.Length > 0)
+                            {
+                                for (int i=1; i<splitString.Length; i++)
+                                {
+                                    atStr = splitString[i] + (i > 1 && i < splitString.Length ? " " : "");
+                                }
+                            }
+                            //thanks xml for not letting me use < or > in attributes :(
```

Comment:
How about using InnerXML for this? Something like `<Conditional>variable>=5</Conditional>`

Reply:
I'm not sure even that'll work, honestly.